examples/demo/src/components/CodePreview.js

```diff
@@ -16,7 +31,10 @@ export default function CodePreview({ code, ...props }) {
               ))}
             </div>
           ))}
-          <CopyButton code={code} />
+          <div className="copy-container">
+            <CopyLang lang={lang} setLang={setLang} />
```

Comment:
I think that "copy-container" and "CopyLang" should be named "snippet-container" and "SnippetLang"; their primary role is to present a code snippet, not be copied :)